### PR TITLE
tests for PTO reqs and Perf revs

### PR DESCRIPTION
**Pull request by Shreyans0102**

### CODE REVIEW

**Review by apourgol — APPROVED** [review]
Looks good!

File: server/routes/performanceReviews.test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    });
+
+
+    test('PUT correctly handles error when updating invalid PerformanceReview', async () => {
+        const res = await request.put('/PerformanceReview/6272079072f180231a71bd31') 
+            .send({
+                type: "assignedTrainings",
+                status: "done",
+                recipient: "John Cena",
+                recipeint_id: "youcantseeme123",
```
Comment (dfsantos-source) [inline server/routes/performanceReviews.test.js:106]: typo here

File: server/routes/PTORequests.test.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        expect(res.status).toBe(200);
+    });
+
+
+    test('POST correctly handles error when posting invalid request', async () => {
+        const res = await request.post('/PTORequests') 
+            .send({
+                type: "assignedTrainings",
+                recipient: "John Cena",
+                recipeint_id: "youcantseeme123",
```
Comment (dfsantos-source) [inline server/routes/PTORequests.test.js:58]: here typo

File: server/routes/PTORequests.test.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    });
+
+
+    test('PUT correctly handles error when updating invalid PTO request', async () => {
+        const res = await request.put('/PTORequests/6272079072f180231a71bd31') 
+            .send({
+                type: "assignedTrainings",
+                status: "done",
+                recipient: "John Cena",
+                recipeint_id: "youcantseeme123",
```
Comment (dfsantos-source) [inline server/routes/PTORequests.test.js:102]: here

File: server/routes/performanceReviews.test.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        expect(res.status).toBe(200);
+    });
+
+
+    test('POST correctly handles error when posting invalid review', async () => {
+        const res = await request.post('/PerformanceReview') 
+            .send({
+                type: "assignedTrainings",
+                recipient: "John Cena",
+                recipeint_id: "youcantseeme123",
```
Comment (dfsantos-source) [inline server/routes/performanceReviews.test.js:54]: ^

**Review by dfsantos-source — APPROVED** [review]

**Comment by dfsantos-source** [comment]
looks good!

**Pull request merged**